pkgs/os-specific/windows/mingw-w64/default.nix

```diff
@@ -14,7 +14,8 @@ let
     then [ "--disable-lib32" "--enable-lib64" ]
     else [];
 
-in stdenv.mkDerivation {
+  crt = stdenv.hostPlatform.libc;
+in assert crt == "ucrt" || crt == "msvcrt"; stdenv.mkDerivation {
```

Comment:
I would instead do
```nix
meta.broken = !(lib.elem crt [ "msvcrt" "ucrt" ]);
```
for each of the `asset`s.

Reply:
Done.